client/interfaces.go

```diff
@@ -6,6 +6,7 @@ import (
 
 	"github.com/gorilla/websocket"
 
+	"github.com/lxc/lxd/lxd/metrics"
```

Comment:
`client` package isn't allowed to import from `lxd/`, anyway, as mentioned below this entire commit should be dropped and we should just use `RawQuery` from `driver_qemu`.